src/components/forms/Form.ts

```diff
@@ -0,0 +1,57 @@
+import './common.scss';
+
+import {Block} from '../../modules/Block';
+
+type IncomingProps = {
+  name: string;
+  action?: string;
```

Comment:
```suggestion
  action?: "GET" | "PUT" | "POST | "DELETE";
```
Тогда если мы (или наш коллега) случайно присвоит некорректную строку этому свойству, TS выдаст предупреждение